src/core.ts

```diff
@@ -36,15 +36,15 @@ export interface JsonFormsServiceConstructable {
  * @constructor
  */
 export const JsonFormsServiceElement = (config) => (cls: JsonFormsServiceConstructable) => {
-  JsonFormsHolder.jsonFormsServices.push(cls);
+  JsonForms.jsonFormsServices.push(cls);
 };
 
 /**
  * Global JSONForms object that holds services and registries.
  */
-export class JsonFormsHolder {
+export class JsonForms {
```

Comment:
JsonForms is also not a perfect name. Is this maybe a JsonFormsConfiguration?
But as I don't have a better name for this, I'm fine with it, we just should think about how to name the global variable that is created by webpack